app/views/business_mailer/subscribed.html.erb

```diff
@@ -0,0 +1,14 @@
+<p><%= @business.first_name %>,</p>
```

Comment:
```suggestion
<p>Hey <%= @business.first_name %>,</p>
```

Reply:
Also, let's remove the blank lines between `<p>` tags to match the other emails.